README.md

```diff
@@ -57,63 +57,64 @@ You can also run the samples in Docker (see below).
 
 1. Update `Startup.cs`'s `ConfigureDevelopmentServices` method as follows:
 
-```
-        public void ConfigureDevelopmentServices(IServiceCollection services)
-        {
-            // use in-memory database
-            //ConfigureTestingServices(services);
+    ```csharp
```

Comment:
Without this being intended:

- the next item will be visible as "1" instead of "2" as it will be considered a completely new list, rather than a new list item in the same list.
- the visual appearance will look bad without the indentation